### PR TITLE
Added option to disable virtualbox vbfs mount of users home directory

### PR DESCRIPTION
**Pull request by rawkode**

### CODE REVIEW

**Comment by ehazlett** [comment]
Thanks for the contribution!  It's a minor nit but do you think we could rename this to something like `virtualbox-no-share`.  I think the `vbfs` gets a little implementation specific.

It also needs a rebase.  Thanks!

**Comment by rawkode** [comment]
No problem, I'll update. Thanks

**Comment by Oliboy50** [comment]
Nice feature here.
I was wondering recently that I should use another file sharing system to speed up my development processes... But ended up to give up because there was the default sharing VBFS stuff anyway.
Thanks :)

File: drivers/virtualbox/virtualbox.go [hunk 1]
```diff
@@ -300,7 +306,7 @@ func (d *Driver) Create() error {
 		return err
 	}
 
-	if shareDir != "" {
+	if shareDir != "" && d.NoVboxFsShare == false {
```
Comment (nathanleclaire) [inline drivers/virtualbox/virtualbox.go]: Nit:  This should probably read, `if shareDir != "" && !d.NoVBoxFsShare`

**Comment by nathanleclaire** [comment]
Looks pretty good to me, needs a rebase.

**Comment by rawkode** [comment]
Updated as per requests :+1:

**Comment by nathanleclaire** [comment]
LGTM

**Comment by ehazlett** [comment]
LGTM

/cc @moxiegirl

**Comment by moxiegirl** [comment]
Thanks @rawkode LGTM to me --- needs rebase and squash as Evan noticed.  How to here:

http://docs.docker.com/project/create-pr/#rebase-your-branch

**Comment by rawkode** [comment]
@moxiegirl rebased and squashed :smile:

**Comment by moxiegirl** [comment]
@rawkode thank you sir.  nice job there. :star2:

**Pull request merged**